VERSION

```diff
@@ -1 +1 @@
-v1.90.8
\ No newline at end of file
+v1.90.8
```

Comment:
can you remove this change (adding the newline)?

Reply:
Sure